apps/electron/main.js

```diff
@@ -30,21 +30,23 @@ let splash;
 let tray = null;
 
 async function startBackend() {
-  await (async () => {
-    // in dev mode, we expect both UI and server to be running
-    if (!isProduction) {
-      return;
-    }
+  // in dev mode, we expect both UI and server to be running
+  if (!isProduction) {
+    return;
+  }
 
-    const ontimeServer = require(nodePath);
-    const { initAssets, startServer, startOSCServer, startIntegrations } = ontimeServer;
+  const ontimeServer = require(nodePath);
+  const { initAssets, startServer, startOSCServer, startIntegrations } = ontimeServer;
 
-    await initAssets();
+  await initAssets();
 
-    loaded = await startServer();
-    await startOSCServer();
-    await startIntegrations();
-  })();
+  const result = await startServer();
+  loaded = result.message;
+
+  await startOSCServer();
+  await startIntegrations();
+
+  return result.serverPort;
```

Comment:
interesting, the IIFE was to workaround the usage of esm module in a cjs environment. curious to see if this still works

Reply:
Hmm... Interesting, I see no reason for this to not work, but I am not to able to test it even because for some reason the `build-win` command is failing.

Reply:
A-ha

It probably fails because of the missing sentry key

You can run `pnpm build:local` on the frontend to fix this